transform/meta_err_test.go

```diff
@@ -34,6 +34,26 @@ var metaErrTests = []struct {
 			[]byte(`{"a":"b"}`),
 		},
 	},
+	{
+		"error_messages",
+		config.Config{
+			Settings: map[string]interface{}{
+				"transform": config.Config{
+					Settings: map[string]interface{}{
+						"message": "test error",
+					},
+					Type: "utility_err",
+				},
+				"error_messages": []string{
+					"test error",
```

Comment:
tests: we should consider adding an additional test with regular expressions in the `error _messages`.

Reply:
Do you mean a regular expression with special characters? These _are_ all regular expressions.

Reply:
Yeah I meant with special characters such as wildcard globs. This way any future refactor or changes that attempt another approach like `strings.Contains(err.Error(), error_message)` to do the matching and regress

Reply:
Ah yep, that's a good idea.